config/config.go

```diff
@@ -679,6 +680,7 @@ type EC2SDConfig struct {
 	SecretKey       string   `yaml:"secret_key,omitempty"`
 	RefreshInterval Duration `yaml:"refresh_interval,omitempty"`
 	Port            int      `yaml:"port"`
+	SubnetSeparator string   `yaml:"subnet_separator,omitempty"`
```

Comment:
Ha, remembered correctly after all – thanks for double-checking.

It was set to comma to be different from relabeling on purpose so you can combine both without ambiguity. Arguably, for a list of similar things a comma is a good fit, while the semicolon is a good fit for relabeling where we concatenate items of different meaning.

So let's stick with the comma.